src/alto-ui/helpers/number.js

```diff
@@ -34,11 +34,13 @@ export const parse = (value, locale, precision = 0, { min, max } = {}) => {
             .replace(/(.+)-/g, '$1')
         );
 
-  return compose(
-    round(precision),
-    x => (min || min === 0 ? Math.max(min, x) : x),
-    x => (max || max === 0 ? Math.min(max, x) : x)
-  )(num);
+  return percent
+    ? num / 100
+    : compose(
+        round(precision),
+        x => (min || min === 0 ? Math.max(min, x) : x),
+        x => (max || max === 0 ? Math.min(max, x) : x)
+      )(num);
```

Comment:
you need to support min/max and precision. Best idea is to plug a new function in the compose pipe.

Reply:
changed :)